molecule/docker_fedora/prepare.yml

```diff
@@ -0,0 +1 @@
+../openstack/prepare.yml
```

Comment:
I don't think it really matters, but it seems weird to link to a file in the openstack scenario rather than do the "shared" scenario dir and link all the prepare books to that.

Reply:
This role appears to pre-date our discovery of that possibility (it was one of our earliest roles and was adapted from a role that I had in my personal projects). The docker folder already linked to the openstack folder for both `prepare.yml` and `playbook.yml`. I just mimicked that. We should probably open a followup issue to go back and retrofit this role with a `shared` folder.

Reply:
Captured as #4

Reply:
This looks good to me